libraries/config/src/config.cpp

```diff
@@ -144,6 +146,11 @@ FullNodeConfig::FullNodeConfig(Json::Value const &string_or_object, Json::Value
     network.network_boot_nodes.push_back(node);
   }
 
+  is_light_node = getConfigDataAsBoolean(root, {"is_light_node"}, true, false);
+  if (is_light_node) {
+    light_node_history = getConfigDataAsUInt(root, {"light_node_history"}, true, kDefaultLightNodeHistory);
```

Comment:
You removed light_node_history in config file

Reply:
yeah I think it was because of my comment with flag. So the idea with flag is that most people, who will run light node wont really configure light_node_history so it is much more user friendly to have the light_node flag, which they can set to true/false and use default value for light_node_history.

But we still can and probably should have also this parameter for setting the light_node_history -> Some advanced users might run some kind of application where they need full data for the past N days for example so they might set this history to some bigger number as the default value we use...

Reply:
this is added as an optional parameter which is not present in the default config files but it is supported, I think this kind of more advanced params do not need to be in the config file but they can be added manually if advanced user wants to add them.